README.md

```diff
@@ -120,14 +103,6 @@ end
 $ bin/rails s
 ```
 
-_For React Apps_
-
-In a separate terminal window/tab
-
-```sh
-$ bin/yarn start
-```
-
```

Comment:
If we do bring back the containers, then most, if not all, of this README change we may want to revert. Much if it is informational/general knowledge on how to run rails w/react, rather than specific to gnarails, but I'm ok with this being a place to share that documentation.